### PR TITLE
Modify eianalytics_dashboard.json by adding generated widgets

### PR DESCRIPTION
**Pull request by isharailanga**
## Purpose
This PR is to modify the  `ei-analytics-dashboards.json` by adding generated widgets

### CODE REVIEW

File: features/ei-analytics-feature/org.wso2.analytics.solutions.ei.analytics.feature/src/main/resources/dashboards/ei-anlaytics-dashboard.json [hunk 1]
```diff
@@ -2363,7 +2363,1177 @@
         "hasViewerPermission": false
```
Comment (lasanthaS) [inline features/ei-analytics-feature/org.wso2.analytics.solutions.ei.analytics.feature/src/main/resources/dashboards/ei-anlaytics-dashboard.json]: Shall we rename the JSON to `eianalytics.json`?

Comment (lasanthaS) [inline features/ei-analytics-feature/org.wso2.analytics.solutions.ei.analytics.feature/src/main/resources/dashboards/ei-anlaytics-dashboard.json]: Shall we rename the JSON to `eianalytics.json`?

Comment (lasanthaS) [inline features/ei-analytics-feature/org.wso2.analytics.solutions.ei.analytics.feature/src/main/resources/dashboards/ei-anlaytics-dashboard.json]: Shall we modify the JSON name to `eianalytics.json`?

Comment (isharailanga) [reply features/ei-analytics-feature/org.wso2.analytics.solutions.ei.analytics.feature/src/main/resources/dashboards/ei-anlaytics-dashboard.json]: Modified at a3796ee2ff05169450bd135a2b5f3b9d6adef228

**Pull request merged**